lib/grape/validations/autoload.rb

```diff
@@ -0,0 +1,52 @@
+module Grape
+  module Validations
+    class Autoload
+      DEFAULT_PATH = %w(api validators)
```

Comment:
Having a default path `validators` makes sense to me, but `api` feels too specific.